app-api/src/test/java/org/openmrs/eip/app/route/sender/DbSyncRouteTest.java

```diff
@@ -49,6 +57,8 @@ public void shouldCreateAndSaveTheSenderSyncMessageForTheEvent() {
 		assertEquals(requestUuid, msg.getRequestUuid());
 		assertNotNull(msg.getDateCreated());
 		assertNotNull(msg.getMessageUuid());
+		assertNull(msg.getDateSent());
+		assertNotNull(msg.getEventDate());
```

Comment:
You should check that debeziumEvent.dateCreated matches eventDate on the sync message